src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -225,7 +223,7 @@ private void createPathwayEdgesAndAddThemToGraph(Graph graph) {
                         fromVertex,
                         toVertex,
                         pathway.getId(),
-                        Optional.ofNullable(pathway.getName()).map(NonLocalizedString::new).orElse((NonLocalizedString)PathwayEdge.DEFAULT_NAME),
+                        NonLocalizedString.ofNullable(pathway.getName()),
```

Comment:
Nice, I like the name  - I would follow old "bad" habits and use "create", but "ofNullable" is much short and clear.

Reply:
I got the name from `Optional.ofNullable`.